index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+
+    <!-- <header> -->
+      <nav id="navbar">
+        <a class="nav" href="#" >  About</a>
+        <a  class="nav" id="profile-link"  href="https://github.com/Azonkeu" target="_blank">  Work</a>
+        <a class="nav" href="#" >  Contact</a>
+
+
+      </nav>
+    <!-- </header> -->
+      <section id="welcome-section">
+        <h1>Hey I am Mimic</h1>
+        <p>a web developer</p>
+      </section>
+      
+    
+    <main>
+      <section id="projects">
+        <h2 class="project-tile">These are some of my projects</h2>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank">Tribute Page</a>
+        </div>
```

Comment:
Please use a `ul` with `li`. The semantic markup will improve accessibility for users that require screen-readers.